hepcrawl/parsers/crossref.py

```diff
@@ -0,0 +1,351 @@
+    def journal_title(self):
+        return get_value(self.record,"container-title[0]")
+
+    @property
+    def artid(self):
+        return self.record.get("article-number")
+
+    @property
+    def journal_issue(self):
+        return self.record.get("issue")
+
+    @property
+    def journal_volume(self):
+        return self.record.get("volume")
+
+    @property
+    def year(self):
+        date = get_value(self.record,"issued.date-parts[0][0]")
+
+        return date
+
+    @property
+    def page_start(self):
+        pages = self.record.get("page")
+
+        if pages:
+            return pages.split('-')[0]
+        else:
+            return None
+
+    @property
+    def page_end(self):
+        pages = self.record.get("page")
+
+        if pages and '-' in pages:
+            return pages.split('-')[1]
+        else:
+            return None
+
+    @staticmethod
+    def get_author_name(author_key):
+        """Extract an author's name."""
+        author_name = ', '.join([author_key.get("family"), author_key.get("given")])
+
+        return author_name
+
+    @staticmethod
+    def get_author_affiliations(author_key):
+        """Extract an author's affiliations."""
+        affiliations = force_list(author_key.get("affiliation"))
+
+        auth_aff = [affiliation.get('name') for affiliation in affiliations]
+
+        return auth_aff
+
+
+    def get_author_orcid(self, author_key):
+        """Extract an author's orcid."""
+        orcid_value = author_key.get('ORCID')
+
+        return [('ORCID', orcid_value)]
+
+
+    def get_author(self, author_key):
+        """Extract one author.
+
+        Args:
+            author_key(dict): a dictionary on a single author.
+
+        Returns:
+            dict: the parsed author, conforming to the Inspire schema.
+        """
+        author_name = self.get_author_name(author_key)
+        affiliations = self.get_author_affiliations(author_key)
+        orcid = self.get_author_orcid(author_key)
+
+        return self.builder.make_author(author_name, raw_affiliations=affiliations, ids=orcid)
+
+    @property
+    def authors(self):
+        authors_key = self.record.get("author")
+        authors = [self.get_author(author) for author in force_list(authors_key)]
+
+        return authors
+
+    @property
+    def license(self):
+        license_keys = self.record.get("license")
+        licenses = [self.get_license(license) for license in force_list(license_keys)]
+
+        return licenses
+
+    def get_license(self, license_key):
+        """Extract one license.
+
+        Args:
+            license_key(dict): a dictionary on a single license.
+
+        Returns:
+            dict: the parsed license, conforming to the Inspire schema.
+        """
+        license = {
+            'imposing': self.publisher,
+            'material': self.material,
+            'url': self.get_license_url(license_key),
+        }
+
+        return license
+
+    def get_license_url(seld, license_key):
```

Comment:
`self` instead of `seld`, but you don't need the `self` here, so should be decorated with `@staticmethod`.